drivers/jsch/pom.xml

```diff
@@ -90,12 +90,12 @@
     <dependency>
       <groupId>com.jcraft</groupId>
       <artifactId>jsch.agentproxy.jsch</artifactId>
-      <version>0.0.8</version>
+      <version>0.0.9</version>
     </dependency>
     <dependency>
       <groupId>com.jcraft</groupId>
       <artifactId>jsch.agentproxy.connector-factory</artifactId>
-      <version>0.0.8</version>
+      <version>0.0.9</version>
```

Comment:
I think one or the other is required. You can see earlier versions had the exclusion here but not below. If you don't have either exclusion you get the duplicate class issue.